pyrefinebio/sample.py

```diff
@@ -119,61 +119,81 @@ def experiments(self, value):
 
     @classmethod
     def get(cls, accession_code):
-        """Retrieve a sample based on its accession code.
+        """Retrieve a Sample based on its accession code.
 
         returns: Sample
 
         parameters:
 
-            accession_code (str): The accession code for the sample to be retrieved.
+            accession_code (str): The accession code for the Sample to be retrieved.
         """
         result = get_by_endpoint("samples/" + accession_code)
         return cls(**result)
 
     @classmethod
     def search(cls, **kwargs):
-        """Retrieve a list of samples based on various filters
+        """Retrieve a list of Samples based on various filters
 
-        returns: list of samples.
+        returns: list of Sample
 
         Parameters:
-            ordering (str):                  which field to use when ordering the results
-            title (str):
-            organism (str):
-            source_database (str):
-            source_archive_url (str):
-            has_raw (str):
-            platform_name (str):
-            technology (str):
-            manufacturer (str):
-            sex (str):
-            age (number):
-            specimen_part (str):
-            genotype (str):
-            disease (str):
-            disease_stage (str):
+            ordering (str): which field to use when ordering the results
+
+            title (str): filter based on the Sample's title
+
+            organism (str): filter based on the Organism that the Sample was taken from
+
+            source_database (str): filter based on the database that the Sample was taken from
+
+            source_archive_url (str): filter based on Sample's source url
+
+            has_raw (bool): filter based on if the Sample had raw data available in the source database
+
+            platform_name (str): filter based on the name of the Platform that was used to create the Sample
+
+            technology (str): filter based on the technology that was used to make the Sample
+
+            manufacturer (str): filter based on the manufacturer of the technology that was used to create the Sample
+
+            sex (str): filter based on the sex of the subject that the Sample was taken from
```

Comment:
One reason I want to get away from the current phrasing is because we don't really have any idea what is going to be in this field! It's whatever the submitted included for upload and will likely depend on the context of the experiment. Ideally, most of the time "the sex of the subject that the Sample was taken from" is correct. I find the use of the term "subject" seems to imply that these are human primary samples and the origin of the samples could be a mouse cell line, for instance. 

```suggestion
            sex (str): filter based on the sex information provided by the submitter
```

Reply:
I'll add that I'm not totally sold on the "provided by the submitter" because it could be a little unclear. When I say submitter, I mean the folks that submitted to the government run repository we got the sample from and this could be taken to mean someone submitted this to refine.bio. Would love to hear from other people!

Reply:
Maybe we could say `original submitter`? Idk, I think either is fine.